### DISCUSSION TITLE
Table 'roles' already exists

### DISCUSSION THREAD

**Discussion by nothingbutboring** [orchidsoftware/platform#2112] [Ideas]
Hi, I tried to install package and I have a problem with tables conflicting.

I use spatie/laravel-permission for roles, orchid use same table name for that, I think it would be great idea to prefix orchid tables to avoid problems in future

**Comment by RaptorGH** [comment]
Im tried it now ((

**Comment by muzhawir** [comment]
Any updates on this issue? I am having the same problem, an existing table conflict with the orchid `roles` table.
